main.py

```diff
@@ -0,0 +1,30 @@
+from sense_hat import SenseHat
+import time
+import sensible_colours
+from ghost_blinky import blinky_frames
+
+
+def fill_pixels(the_sense_hat, the_animation):
+    the_sense_hat.low_light = True
+
+    num_frames = len(the_animation)
+    count = 0
+    while True:
+        pixels = the_animation[count % num_frames]
+        the_sense_hat.set_pixels(pixels)
+        time.sleep(1 / 4)
```

Comment:
That's a weird way to create a magic value. What's the purpose?

Reply:
Using 1/4 was to provide base to create a "frames per second" variable to allow the animation speed to be varied from one location, or via command line argument, or similar in a future version. The 4 would be replaced by the variable,

Reply:
Ah yes, that makes sense. It might have helped to create a variable "frames_per_second" right away, as that would explain the purpose. :)

Reply:
**Note:**

- 4 frames per second = 0.25 seconds between frames
- 10 frames per second = 0.1 seconds between frames
- Persistence of vision starts to occur at just over 12 frames per second.